src/Status.cxx

```diff
@@ -0,0 +1,154 @@
+#include <cmath>
+
+#include <rapidjson/document.h>
+
+#include "Status.hpp"
+
+double average(const double &sum, const double &N) { return sum / N; }
+
+double standard_deviation(const double &sum, const double &sum_squared,
+                          const double &N) {
+  auto variance = (N * sum_squared - sum * sum) / (N * (N - 1));
```

Comment:
Is there a reason for the additional `N` in numerator and denominator?

Reply:
Just traded one division for two multiplication. I was trying to understand if one of the two solution can be more stable when ```sum_squared``` \approx ```sum * sum```, but found the two equivalent. I can switch to the other form if it looks more clear